src/test_utils.rs

```diff
@@ -935,9 +935,9 @@ impl TestContext {
     /// so may create a key-contact with a fingerprint
     /// but without the key.
     pub async fn get_chat(&self, other: &TestContext) -> Chat {
-        let contact = self.add_or_lookup_contact_id(other).await;
+        let contact = self.add_or_lookup_contact_no_key(other).await;
 
-        let chat_id = ChatIdBlocked::lookup_by_contact(&self.ctx, contact)
+        let chat_id = ChatIdBlocked::lookup_by_contact(&self.ctx, contact.id)
```

Comment:
This change is necessary in order to make my test pass.

I think that it was unexpected before that a function called `get_chat()` had the side-effect of transferring name and pubkey of the contact.

Reply:
Since we don't need a contact, but only the ID, maybe add `add_or_lookup_contact_id_no_key` and use it here and in `add_or_lookup_contact_no_key`?